src/number_theory/arithmetic_function.lean

```diff
@@ -732,6 +732,25 @@ begin
   { rcases h with h | h; simp [h] }
 end
 
+lemma is_multiplicative_moebius : is_multiplicative μ :=
+⟨by simp, begin
+  intros m n hmn,
+  by_cases hm_sq : squarefree m,
+  { by_cases hn_sq : squarefree n,
+    { simp [hm_sq, hn_sq, (squarefree_mul hmn).mp ⟨hm_sq, hn_sq⟩],
+      rw arithmetic_function.card_factors_mul (ne_zero_of_squarefree hm_sq)
+        (ne_zero_of_squarefree hn_sq),
+      rw pow_add, },
+    { have : ¬ squarefree (m * n),
+        by_contradiction H,
+        exact hn_sq ((squarefree_mul hmn).mpr H).right,
+      simp [this, hn_sq], }, },
+  have : ¬ squarefree (m * n),
+    by_contradiction H,
+    exact hm_sq ((squarefree_mul hmn).mpr H).left,
```

Comment:
Similarly to above; mathlib style is to use `{}` not indents, as `{}` can't lie about where the goals are:
```suggestion
  have : ¬ squarefree (m * n),
  { by_contradiction H,
    exact hm_sq ((squarefree_mul hmn).mpr H).left },
```

Reply:
Thanks! Trying to learn the style. Is there a linter available?

Reply:
all the linters that exist run automatically; you can run them by hand by putting `#lint` on a blank line. There is a style guide, though: https://leanprover-community.github.io/contribute/style.html

Reply:
Thanks so much!